DOCS: Make better clear refunds can be done via MCP (#1241)
MultiSafepay/docs: content/integrations/ecommerce-integrations/magento2/faq/refunding-magento2.md

You can also refund from your [MultiSafepay Control](https://merchant.multisafepay.com)

{{< blue-notice >}}

> If you have just upgraded from the deprecated plugin to the new one, please note that refunding the orders with a payment method from the deprecated plugin is not possible inside Magento, because the payment method is not available anymore. These orders can be refunded via [MultiSafepay Control](https://merchant.multisafepay.com) instead.

Reviewer: ```suggestion
If you have upgraded from the deprecated plugin to the new one, refunds of orders with payment methods from the deprecated plugin are not available inside Magento. These orders can be refunded via [MultiSafepay Control](https://merchant.multisafepay.com) instead.
```

{{< /blue-notice >}}

{{< blue-notice >}}

> Refunding from the Magento 2 backend is disabled when the order has a Fooman Surcharge. It is still possible to refund those orders through [MultiSafepay Control](https://merchant.multisafepay.com)

Reviewer: ```suggestion
Refunding from the Magento 2 backend is disabled when the order has a Fooman Surcharge. It is still possible to refund those orders through [MultiSafepay Control](https://merchant.multisafepay.com)
```

{{< /blue-notice >}}

> If you have more questions about refunds, please check out our [FAQ on refunds](/faq/refunds).

Reviewer: ```suggestion
For more information about refunds, please check out our [documentation on refunds](/faq/refunds).
```
